tequilapi/contract/custom_json_types.go

```diff
@@ -0,0 +1,36 @@
+package contract
+
+import (
+	"fmt"
+	"math/big"
+)
+
+type BigInt struct {
+	big.Int
```

Comment:
If you use `*big.Int` you'll lose on some of the cool things it [implements](https://golang.org/pkg/math/big/): https://play.golang.org/p/0RrQwVdgGlb